oneview/resource_server_profile.go

```diff
@@ -523,7 +508,7 @@ func resourceServerProfile() *schema.Resource {
 												},
 												"drive_number": {
 													Type:     schema.TypeInt,
-													Optional: true,
+													Computed: true,
 												},
 												"drive_technology": {
 													Type:     schema.TypeString,
```

Comment:
if the local storage is configured from SPT and apply to SP, have we validated? i mean are we not getting any idempotency issues